src/_includes/layouts/base.njk

```diff
@@ -30,6 +36,52 @@
             </article>
         </main>
         {% include "partials/global/footer.njk" %}
-        {% uioInit %}
+        {# {% uioInit %} #}
+        <script>
+        fluid.defaults("fluid.uiOptions.multilingual", {
+            gradeNames: ["fluid.uiOptions"],
+            distributeOptions: {
+                source: "{that}.options.locale",
+                target: "{that messageLoader}.options.resourceOptions.locale"
+            },
```

Comment:
You could remove this options distributions entirely if you set the `defaultLocale` In the `prefsEditorLoader` options. 

This would replace line 82-83 below.

```javascript
prefsEditorLoader: {
    defaultLocale: "{{ lang | truncate(2, true, '') }}",
    direction: "{{ config.languages | getSiblingValue("code", lang, "dir") }}"
}
```